modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyQuery.jsx

```diff
@@ -48,20 +49,30 @@ const MAX_RESULTS = 10;
 //  placeholder: String to display as the input element's placeholder
 //  value: String to use as the input element value
 //  onChange: Callback in term input change event
+//  allowTermSelection: Boolean enabler for term selection from vocabulary tree browser
+//  initialSelection: Existing answers
+//  onRemoveOption: Function to remove added answer
 //
 function VocabularyQuery(props) {
   const { clearOnClick, onClick, focusAfterSelecting, disabled, variant, isNested, placeholder,
-    value, questionDefinition, onChange, classes } = props;
-
+    value, questionDefinition, onChange, allowTermSelection, initialSelection, onRemoveOption, classes } = props;
   const [suggestions, setSuggestions] = useState([]);
   const [suggestionsLoading, setSuggestionsLoading] = useState(false);
   const [suggestionsVisible, setSuggestionsVisible] = useState(false);
   const [lookupTimer, setLookupTimer] = useState(null);
+  const maxAnswers = questionDefinition?.maxAnswers;
 
   // Holds term path on dropdown info button click
   const [termPath, setTermPath] = useState("");
 
-  const [inputValue, setInputValue] = useState(value);
+  // Checks whether path is listed in the default answer options in the question definition
+  let isDefaultOption = (path) => {
+    return Object.values(props.questionDefinition)
+          .filter(value => value['jcr:primaryType'] == 'lfs:AnswerOption' && value.value === path)
+          .length > 0;
```

Comment:
I find this easier to read:
```suggestion
          .find(value => value['jcr:primaryType'] == 'lfs:AnswerOption' && value.value === path)
```